gen/xyz/openbmc_project/User/meson.build

```diff
@@ -44,7 +44,37 @@ generated_others += custom_target(
     ],
 )
 
+subdir('GoogleAuthenticator')
+generated_others += custom_target(
+    'xyz/openbmc_project/User/GoogleAuthenticator__markdown'.underscorify(),
+    input: [ '../../../../yaml/xyz/openbmc_project/User/GoogleAuthenticator.interface.yaml',  ],
+    output: [ 'GoogleAuthenticator.md' ],
+    depend_files: sdbusplusplus_depfiles,
+    command: [
+        sdbuspp_gen_meson_prog, '--command', 'markdown',
+        '--output', meson.current_build_dir(),
+        '--tool', sdbusplusplus_prog,
+        '--directory', meson.current_source_dir() / '../../../../yaml',
+        'xyz/openbmc_project/User/GoogleAuthenticator',
+    ],
+)
+
 subdir('Ldap')
+subdir('MFABypass')
```

Comment:
Thought this should be in chronological order, otherwise, there might be some CI issues.
But CI has passed anyways.

Reply:
yeah CI passed